app/assets/javascripts/kuroko2/job_instances.js

```diff
@@ -45,6 +48,7 @@ jQuery(function ($) {
         tbody.data('last-log-id', data.logs[data.logs.length - 1].id);
       }
 
+      $('#logs table').data("reload", data.reload);
```

Comment:
It is used the below line.
https://github.com/eisuke/kuroko2/blob/f9db129f58e3d045db2356f0380fe27a69d1153c/app/assets/javascripts/kuroko2/job_instances.js#L90